stories/vrc-stories/Layout.js

```diff
@@ -13,24 +14,140 @@ const Container = ({ children }) => (
   <div style={{ maxWidth: '600px', display: 'flex', flexGrow: '1' }}>{children}</div>
 );
 
-storiesOf('Layout', module).lokiSkip('default', () => (
-  <Container>
-    <Layout>
-      <Header company={text('Company Name', 'Virtru')} logo={text('Company Logo', undefined)} title={text('Header Title', 'My App')} />
-      Content
-    </Layout>
-  </Container>
-)).lokiSkip('default w/o nav', () => (
-  <Container>
-    <Layout>
-      <Header company={text('Company Name', 'Virtru')} logo={text('Company Logo', undefined)} title={text('Header Title', 'My App')}>
-        <Menu>
-          <MenuItem>One</MenuItem>
-          <MenuItem>Two</MenuItem>
-          <MenuItem>Three</MenuItem>
-        </Menu>
-      </Header>
-      Content
-    </Layout>
-  </Container>
-))
+storiesOf('Layout', module)
+  .lokiSkip('default', () => (
+    <Container>
+      <Layout>Content here</Layout>
+    </Container>
+  ))
+  .lokiSkip('with header', () => (
+    <Container>
+      <Layout>
+        <Header
+          company={text('Company Name', 'Virtru')}
+          logo={text('Company Logo', undefined)}
+          title={text('Header Title', 'My App')}
+        ></Header>
+        Content here
+      </Layout>
+    </Container>
+  ))
+  .lokiSkip('with header and nav', () => (
```

Comment:
Why `lokiSkip` on these 3 stories?

Reply:
Because they are interactive. 🤷‍♂️

Reply:
I assumed we used Loki for all stories, but I should dig in further some time.